wwwapp/templatetags/wwwtags.py

```diff
@@ -12,3 +11,11 @@ def qualified_mark(value):
         return mark_safe('<span class="qualified">✔</span> TAK')
     else:
         return mark_safe('<span class="not-qualified">✘</span> NIE')
+
+@register.filter
+def maybe_not_present(value):
```

Comment:
Different handling of None and empty????????? Please split the functions

Reply:
:+1:, and what is `maybe_not_present` even supposed to mean?